gui/qt/fee_slider.py

```diff
@@ -48,3 +51,15 @@ def update(self):
             self.setValue(pos)
             tooltip = self.get_tooltip(pos, fee_rate)
             self.setToolTip(tooltip)
+
+    # configuraing this as is done is here still required, can't just set range 0,0 to deactivate.
+    # chose to make this a seperate function from update for easier code maintainence
+    def deactivate(self):
+        with self.lock:
```

Comment:
Why using self.lock here?  Isn't all GUI code going to be run in the main thread anyway? Or does code from other threads call this code?

It's harmless to have it just it's interesting to me that a lock is being held in a GUI code method...